lib/snackbar/theme.module.scss

```diff
@@ -34,10 +34,34 @@
 }
 
 :local(.snackbar) {
-  background: $primary-color;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 42em;
+  background: $cool-blue;
   border-radius: 3px;
-  padding: 10px 20px;
+  padding: 0.7em 1.2em;
```

Comment:
make it padding: 0.9em 1.2em and it is good to go.

Reply:
@kaushiknishchay Updated padding in [5994f01](https://github.com/Codebrahma/react-lite-ui/pull/294/commits/5994f01dbd08ff566575a4d7c9f0a3d528e85bd3)